package.json

```diff
@@ -16,9 +15,15 @@
     "./plugin-example-one": {
       "import": "./dist/plugin-example-one/index.mjs",
       "types": "./dist/plugin-example-one/index.d.ts"
+    },
+    "./plugin-dynamic-import-retry": {
+      "import": "./dist/plugin-dynamic-import-retry/index.mjs",
+      "types": "./dist/plugin-dynamic-import-retry/index.d.ts"
```

Comment:
nit: shall we remove the `plugin-` prefix here and in `src`/`dist` folders and expose the plugin as `@kong/vite-plugins/dynamic-import-retry`?

@adamdehaven WDYT?

Reply:
Sounds good to me, but would need to be reflected in the docs as well. 

I originally did this on purpose and likely forgot to switch the format. 

Would you mind doing in a separate PR so it's not related to this plugin?